### PR TITLE
Create the Official BuddyPress Logo document

### PR DESCRIPTION
**Pull request by emaralive**
Create the "Official BuddyPress Logo" document. 

bp-documentation:  https://github.com/buddypress/bp-documentation/issues/5

---
**This Pull Request is for code review only. Please keep all other discussion in the BuddyPress Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the WordPress Core Handbook for more details.**

### CODE REVIEW

File: docs/logo.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<div align="center">
+	<h1>Official BuddyPress Logo</h1>
+</div>
```
Comment (imath) [inline docs/logo.md:3]: We need to use Markdown for the documentation, I can understand it's maybe a bit frustrating as we may not be able to center paragraph or have less control about table rendering etc... But it's a requirement.

```suggestion
# Official BuddyPress Logo
```

File: docs/logo.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<p>Usage of the official BuddyPress logo, e.g., for website or publication, is granted in accordance with the <a href="http://wordpressfoundation.org/trademark-policy/">WordPress trademark policy</a>. The <a href="https://www.exljbris.com/fontin.html">Fontin</a> typeface is used for the official BuddyPress logotype. BuddyPress related Dashicons 
+can be found at <a href="https://developer.wordpress.org/resource/dashicons/#icons-buddicons">Buddicons.</a></p>
```
Comment (imath) [inline docs/logo.md:5]:
```suggestion

Usage of the official BuddyPress logo, e.g., for website or publication, is granted in accordance with the [WordPress trademark policy](http://wordpressfoundation.org/trademark-policy/). The [Fontin](https://www.exljbris.com/fontin.html) typeface is used for the official BuddyPress logotype. BuddyPress related Dashicons can be found at [Buddicons](https://developer.wordpress.org/resource/dashicons/#icons-buddicons).

```

File: docs/logo.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+			<td></td>
+			<td><a href="assets/bp-blue-1024x768.png">PNG (1024 x 768)</a></td>
+		</tr>
+		<tr>
+			<td><a href="assets/bp-light-hi-640x960.png">PNG (640 x 960)</a></td>
+			<td></td>
+			<td><a href="assets/bp-blue-640x960.png">PNG (640 x 960)</a></td>
+		</tr>
+	</table>
+</div>
```
Comment (imath) [inline docs/logo.md:85]: We need a Markdown table.

**Review by imath — CHANGES_REQUESTED** [review]
Hi @emaralive 

First, thank you very much for your work on this PR. You did an amazing job. But as explained into the contributor Docs, writing documentation is done using Markdown, see:
https://github.com/buddypress/buddypress/blob/master/docs/contributor/documentation/README.md#how-to-contribute-to-buddypress-docs

I've added some suggestions, but the tables need more work, so I simply added an example of how I would "convert" it in Markdown tables.

Could you update the logo.md file so that it's written in [Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)?

`+` Could you add a link to this logo.md file in the last bullet point of this page?
https://github.com/buddypress/buddypress/tree/master/docs#readme

File: docs/logo.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+			<td><a href="assets/bp-brandmark.svg">SVG (vector)</a></td>
+			<td></td>
+			<td><a href="assets/bp-combination.svg">SVG (vector)</a></td>
+			<td></td>
+			<td></td>
+			<td></td>
+			<td><a href="assets/bp-mascot.svg">SVG (vector)</a</td>
+		</tr>
+	</table>
+</div>
```
Comment (imath) [inline docs/logo.md:43]: We need a markdown table too, eg:

```md
| Center-aligned | | Center-aligned |
| :--- : |:---:|:---: |
| ![Image](assets/image.png| |![Image](assets/image.png)|
|Text|Text|Text|
```

**Comment by emaralive** [comment]
@imath,

I forced pushed a 2nd commit which has all the changes you requested. I had to resize all the thumbs to fit into the layout. However, there appear to be 2 commits (1st original and 2nd changes requested).  

Not sure if I need to do something to remove the original commit or is this good enough?

**Review by imath — APPROVED** [review]
Hi @emaralive

One word, perfect! Thanks a lot for your changes: it's ready to be SVN committed. I'll do it asap.

**Pull request closed without merging**